src/chargeTask.hpp

```diff
@@ -22,13 +22,28 @@ class ChargeTask : public Task{
     void init(void);
     /**
      * @brief Charges the device, and exits to CLI on command
-    */
+     * 
+     * @return Current status of charge task
```

Comment:
Is the return value the current status?  Please inspect the function implementation.